src/js/components/Form/__tests__/Form-test-uncontrolled.js

```diff
@@ -609,7 +679,6 @@ describe('Form uncontrolled', () => {
       target: { value: 'J' },
     });
     act(() => jest.advanceTimersByTime(1000)); // allow validations to run
-    expect(queryAllByText('must be >1 character')).toHaveLength(1);
```

Comment:
Normally, changing the tests due to build breaks is a recipe for potential breaking changes.
Can you please explain why is this change needed?

Reply:
I moved it to a new test suite name 'validate on change'.
Wanted to keep this test suite only for testing valid prop.

Reply:
I get it.
I have moved the line to a different test case 'validate on change'.

Will keep you updated on why this line was failing in this specific case.

Reply:
// act(() => jest.advanceTimersByTime(1000)); // allow validations to run
    setTimeout(() => {
      expect(queryByText('must be >1 character')).toBe(null);
      expect(valid).toBeFalsy();
    }, 600);

Using setTimeout works, looks like advanceTimersByTime is not the right way here.